packages/admin/admin/src/tabs/TabScrollButton.tsx

```diff
@@ -1,51 +1,58 @@
 import { ChevronLeft, ChevronRight } from "@comet/admin-icons";
-import {
-    ButtonBase,
-    ComponentsOverrides,
-    TabScrollButtonClassKey as MuiTabScrollButtonClassKey,
-    TabScrollButtonProps as MuiTabScrollButtonProps,
-    Theme,
-} from "@mui/material";
-import { createStyles, WithStyles, withStyles } from "@mui/styles";
+import { ButtonBase, ButtonBaseProps, ComponentsOverrides } from "@mui/material";
+import { css, styled, Theme, useThemeProps } from "@mui/material/styles";
 import isMobile from "is-mobile";
 import * as React from "react";
 
-export type TabScrollButtonClassKey = MuiTabScrollButtonClassKey;
+export type TabScrollButtonClassKey = "root" | "vertical" | "disabled";
 
-export interface TabScrollButtonProps extends MuiTabScrollButtonProps {
-    onClick: (event: React.MouseEvent<HTMLElement>) => void;
+type OwnerState = Pick<TabScrollButtonProps, "orientation">;
+
+const Root = styled(ButtonBase, {
+    name: "CometAdminTabScrollButton",
+    slot: "root",
+    overridesResolver({ ownerState }: { ownerState: OwnerState }, styles) {
+        return [styles.root, ownerState.orientation && styles.vertical];
```

Comment:
The `vertical` styles should only be set, when the `orientation` is set to `vertical`

```suggestion
        return [styles.root, ownerState.orientation === "vertical" && styles.vertical];
```